Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -51,8 +61,13 @@ internal final class AnalyticsProvider: AnalyticsProviderProtocol {
     internal let apiClient: APIClientProtocol
     internal let configuration: AnalyticsConfiguration
     internal private(set) var checkoutAttemptId: String?
-    internal var additionalFields: (() -> AdditionalAnalyticsFields)?
     private let uniqueAssetAPIClient: UniqueAssetAPIClient<CheckoutAttemptIdResponse>
+    
+    private var batchTimer: Timer?
+    
+    private var events: [AdyenAnalytics.Event] = []
```

Comment:
maybe make a struct for events + logs + errors (e.g. `AnalyticsDataPoints`) which makes it easier to handle and reset them